src/components/FaresList.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Dispatch, useEffect, useState } from 'react';
+import { createTheme, ThemeProvider } from '@mui/material';
+import MaterialTable from 'material-table';

```

Comment:
Not sure if caused by this, or something else, but the table quite easily gets overwhelmed by clicks and crashes 🙃

Reply:
generally in projects we'll have more specific styling needs so will implement components mostly from scratch - so we would have more control over these sorts of things